docs/source/json-api/index.rst

```diff
@@ -259,12 +259,13 @@ you can also use the ``--query-store-jdbc-config`` CLI flag (deprecated), an exa
 Access Tokens
 =============
 
-The JSON API essentially performs two separate tasks:
+Each request to the HTTP JSON API Service *must* come with an access token, regardless of whether the underlying ledger
+requires it or not. This also includes development setups using an unsecured sandbox. The HTTP JSON API Service *does not*
+hold on to the access token, which will only used to fulfill the request it came along with. The same token will be used
```

Comment:
only used -> only be used

Reply:
Thanks https://github.com/digital-asset/daml/pull/13331